packages/design/src/components/input-number/InputNumber.tsx

```diff
@@ -58,6 +58,12 @@ export interface IInputNumberProps {
      * @param value
      */
     onChange?: (value: number | null) => void;
+
+    /**
+     * Callback when user press enter
+     * @param e
+     */
+    onPressEnter?: (e: React.KeyboardEvent<HTMLInputElement>) => void;
```

Comment:
What's the difference between `onChange` and `onPressEnter`?

Reply:
This is a redundant design that might be used in the future